packages/components/psammead-topic-tags/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,649 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TopicTags should correctly render a single topic for ltr 1`] = `
+.emotion-0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-flex-wrap: wrap;
+  -webkit-flex-wrap: wrap;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+}
+
+.emotion-2 {
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  margin-right: 0.5rem;
+  word-break: break-word;
+}
+
+@media (min-width: 20rem) and (max-width: 37.4375rem) {
+  .emotion-2 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width: 37.5rem) {
+  .emotion-2 {
+    font-size: 0.8125rem;
+    line-height: 1rem;
+  }
+}
+
+.emotion-2 a {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  color: #222222;
+}
+
+.emotion-2 a:hover {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-2 a:visited {
+  color: #6E6E73;
+}
+
+.emotion-2 a>span {
+  background-color: #F2F2F2;
+  padding: 0.375rem 0.4375rem;
+}
+
+.emotion-2:not(:first-of-type) {
+  margin-top: 0.0625rem;
+}
+
+<div>
```

Comment:
So no list when only 1 link? Excellent.

Reply:
Yes, the component renders a `<div>` instead of a `<ul>` if there is only 1 `TopicTag` component within the `TopicTags` component 👍